jobbergate-agent/README.md

```diff
@@ -0,0 +1,63 @@
+# Jobbergate-agent
+
+# Table of contents
+
+- [Project setup](#project-setup)
+  - [Dependencies](#dependencies)
+- [Install the package](#install-the-package)
+- [Setup parameters](#setup-parameters)
+- [Local usage example](#local-usage-exemple)
+
+## Project Setup
+
+### Dependencies
+
+- python3-venv
+
+## Install the package
+
+To install the package from Pypi simply run `pip install jobbergate-agent`.
+
+## Setup parameters
+
+1. Setup dependencies
+  You can use whenever dependency manager you want to. Just run the command below (and the ones following) on behalf of the manager you prefer.
+
+  ```bash
+  make dependencies
+  ```
+
+2. Setup `.env` parameters
+
+  ```bash
+  CLUSTER_AGENT_BASE_API_URL="<base-api-url>"
```

Comment:
Can we change the settings values so that they are prefixed with "JOBBERGATE_AGENT" instead?